inscriptions/ISic000570.xml

```diff
@@ -185,14 +190,21 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#GC">
+                <ab>
+                    <lb n="1"/><expan><abbr>D</abbr><ex>is</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>m</abbr><ex>anibus</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>s</abbr><ex>acrum</ex></expan>
+                    <lb n="2"/><name><expan><abbr>P</abbr><ex>ublio</ex></expan></name> <g ref="#interpunct">·</g> Servili <g ref="#interpunct">·</g> Paphroditi
+                    <lb n="3"/><expan><abbr>v</abbr><ex>ixit</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>an</abbr><ex>is</ex></expan> <g ref="#interpunct">·</g> <num value="26">XXVI</num> <g ref="#interpunct">·</g> <name>Encoipus</name> <g ref="#interpunct">·</g> <expan><abbr>p</abbr><ex>osuit</ex></expan>
```

Comment:
`<expan><abbr>an</abbr><ex>nis</ex></expan>` (you missed an n from annis)

and `<name>Encoipus</name>` should be `<name>Enco<choice><corr>l</corr><sic>e</sic>pus</name>`